style.css

```diff
@@ -0,0 +1,116 @@
+@import url('https://rsms.me/inter/inter.css');
+html { font-family: 'Inter', sans-serif; }
+
+@supports (font-variation-settings: normal) {
+  html {
+    font-family: 'Inter var', sans-serif;
+  }
+}
+
+body {
+  background-color: #e5e5e5;
+  margin: auto;
+  position: relative;
+  width: 375px;
+  height: 6715px;
+}
+
+.global {
+  margin: auto;
+  position: relative;
+  width: 375px;
+  height: 950px;
+  background-color: #fff;
+}
+
+.appbar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+}
+
+.toolbar {
+  position: absolute;
+  top: 44px;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 114px;
+  height: 65px;
+  left: 12px;
+  top: calc(50% - 40px / 2);
+}
+
+.menu {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 35px;
+  left: 294px;
+  top: calc(50% - 40% / 2);
+  border-radius: 2px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 54px;
+  gap: 28px;
+  position: absolute;
+  background-image: url(images/background.svg);
```

Comment:
- [ ] `required`
- [ ] Good job implementing some of the changes
- [ ] the background image has to be moved below the harmburg and the welcome just as the previous reviewer mentioned. 
- [ ] see screenshot below

`your template`
![mob](https://user-images.githubusercontent.com/66230306/186891999-2018c485-91a4-4967-b33b-22d2af801b07.png)




`figma template`
![image](https://user-images.githubusercontent.com/66230306/186891847-916e41a3-6392-467c-bbf0-ba8a732070e2.png)